libs/ui/src/session_time_limit_tracker.tsx

```diff
@@ -0,0 +1,206 @@
+/* istanbul ignore file */
```

Comment:
As I mentioned in the PR description, this PR is large enough as is so I'm going to add tests for this component in a follow-up